### DISCUSSION TITLE
Are there any functions that calculate geometry symmetry class automatically?

### DISCUSSION THREAD

**Discussion by teltim** [rdkit/rdkit#6421] [Q&A] [answered]
There is a function to get symmetry class of moleclue,
When inputing an exsample, I got {(0,), (1, 2, 3)}
How can I translate the output to symmetry class here https://psicode.org/psi4manual/master/psithonmol.html#symmetry

https://rdkit-discuss.narkive.com/5NftjDXG/symmetry-class
def FindSymmetryClasses(m):
    equivs = defaultdict(set)
    matches = m.GetSubstructMatches(m,uniquify=False)
    for match in matches:
        for idx1,idx2 in enumerate(match): equivs[idx1].add(idx2)
    classes = set()
    for s in equivs.values(): classes.add(tuple(s))
    return classes

**Comment by greglandrum** [comment, marked as answer]
Hi @teltim : the RDKit does not have any functionality for point-group symmetry detection.

**Reply by teltim** [reply]
I see, thanks for the reply.
